tests/Block/FeatureMediaBlockServiceTest.php

```diff
@@ -32,7 +32,7 @@ protected function setUp(): void
         $this->galleryManager = $this->prophesize(GalleryManagerInterface::class);
 
         $this->blockService = new FeatureMediaBlockService(
-            'block.service',
+            $this->templating,
```

Comment:
templating twice?

Reply:
The thing is that [`MediaBlockService` expects a `name` and a `EngineInterface`](https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/Block/MediaBlockService.php#L56) and [passing a `name` as string is deprecated](https://github.com/sonata-project/SonataBlockBundle/blob/3.x/src/Block/Service/AbstractBlockService.php#L51)

Reply:
Ok!